kivy/uix/image.py

```diff
@@ -138,18 +145,22 @@ def get_image_ratio(self):
         :class:`~kivy.properties.ColorProperty`.
     '''
 
-    allow_stretch = BooleanProperty(False)
+    allow_stretch = BooleanProperty(False, deprecated=True)
     '''If True, the normalized image size will be maximized to fit in the image
     box. Otherwise, if the box is too tall, the image will not be
     stretched more than 1:1 pixels.
 
     .. versionadded:: 1.0.7
 
+    .. deprecated:: 2.2.0
+        :attr:`allow_stretch` have been deprecated. Please use `fit_mode`
```

Comment:
We're still suggesting the usage of `allow_stretch`, even if is now deprecated:
https://github.com/kivy/kivy/blob/e2c8f0753337cd693ae5b34de0b3bd3a481d725c/kivy/uix/image.py#L35
https://github.com/kivy/kivy/blob/e2c8f0753337cd693ae5b34de0b3bd3a481d725c/kivy/uix/image.py#L83